test/test_remap_files.cpp

```diff
@@ -49,9 +49,9 @@ boost::shared_ptr<T> clone_ptr(boost::shared_ptr<T> const& ptr)
 	return boost::shared_ptr<T>(new T(*ptr));
 }
 
-int peer_disconnects = 0;
+static int peer_disconnects = 0;
```

Comment:
I'm indifferent, will use those. Also I might have broken the remap_files test in one of the Travis builds, will fix that as well.